### PR TITLE
WFLY-15713 Remove redundant null checks in TimerHandleImpl constructor.

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-15713

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@chengfang can you please review once :)

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 1]
```diff
@@ -2284,30 +2284,6 @@
...
-
-    /**
-     * Creates an exception indicating the timer service cannot be null
-     *
-     * @return an {@link IllegalArgumentException} for the error.
-     */
-    @Message(id = 337, value = "Timer service cannot be null")
-    IllegalArgumentException timerServiceIsNull();
-
     /**
```
Comment (chengfang) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:2287]: We should just comment out these methods to make it easy to see their history and not to accidentally reuse these ids.

File: ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerHandleImpl.java [hunk 2]
```diff
@@ -69,16 +69,6 @@
      * @throws IllegalArgumentException If either of the passed parameters is null
      */
     public TimerHandleImpl(final String id, final String timedObjectId, final TimerServiceImpl service) throws IllegalArgumentException {
```
Comment (chengfang) [inline ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerHandleImpl.java:71]: Can you also remove the IllegalArgumentException from method signature, and javadoc?

**Comment by RanabirChakraborty** [comment]
Thanks! did the changes.

**Review by chengfang — APPROVED** [review]

**Pull request merged**